### ISSUE TITLE
Activation Bug

### ISSUE THREAD

**Issue by jbilen** [thomaslindstrom/color-picker#128]
Will not activate with the shortcut keys (i.e. Ctrl+Alt+C) or by right-click and select "Color Picker," if the current position is less than 12 lines from the top.

**Comment by thomaslindstrom** [comment]
Hi, @jbilen. Could you give me some more information on this issue? What version of Atom and `color-picker` are you on? Are you seeing an error message?

**Comment by jbilen** [comment]
Hi,

I'm running Atom 1.0.2 and Color Picker 2.0.10 and in order for the package
to work the line with the color has to be at least 12 lines from the top of
the editor window. I'm also running the Pigments package, so I'm wondering
if there is a conflict between the two?

Regards,

Jeff

**Comment by miclf** [comment]
I have the same issue but, in my case, the color picker is only triggered when my cursor is located in the the lower half of the editor’s height (my editor is ≃ 34 lines tall).

Same version of Atom (1.0.2) and package (2.0.10). But I do *not* have Pigments installed, so it’s probably not related to it.

**Comment by thomaslindstrom** [comment]
Alright... @miclf and @jbilen, what OS are you on? If you try opening the `color-picker` again, and it fails, would try try opening the inspector (`CMD-ALT-I`), then make sure it still doesn't work, then give me the result of pasting this in the console?

```
var e; (e = atom.workspace.getActiveTextEditor()); ("" + e.getVisibleRowRange().join(',') + " - " + e.getLastCursor().getBufferRow());
```

**Comment by maxbook** [comment]
I have the same issue, the cursor position in active text editor is wrong, to resolve replace this line in color-picker/lib/ColorPicker-view.coffee (270)

<pre> _cursorRow = Cursor.getBufferRow()</pre>

By 

<pre>_cursorRow = Editor.getCursorScreenPosition().row</pre>

:D

**Comment by jbilen** [comment]
Perfect! Fixed it for me... Thanks

**Comment by miclf** [comment]
@thomaslindstrom I’m on OS X 10.10.4.

Result for a non-working line: `"4234,4257 - 4222"`
Result for a working one: `"4234,4257 - 4235"`

@maxbook your fix also works for me *but* it introduces a new bug. When opening the color picker, a new cursor is added in my file, exactly 19 lines under the line from which I opened the color picker. Weird.

**Comment by maxbook** [comment]
Yes, i have the same bug when my code is folded, but only once. Strange...  I see that ;)

**Comment by pavloko** [comment]
Experiencing the same issue as @miclf .  

The color-picker activation works, but it now seems to add a second cursor, a number of lines below the actual color code, the deeper you in the document the more lines cursor jumps.

**Comment by maxbook** [comment]
everything is ok , I had corrected the problem rather quickly ^^ now it's fixed

**Comment by thomaslindstrom** [comment]
Merged @maxbook's pull request. This should be fixed in `v2.0.11`!
